### PR TITLE
Fix bouns tile

### PR DESCRIPTION
**Pull request by frudolph77**
When a track tile get placed the info about the bonus tile gets lost, added a small fix to keep that info.
Also added a star to highlight the bonus tile when no track is placed.

### CODE REVIEW

File: src/Cell.css [hunk 1]
```diff
@@ -53,7 +53,7 @@
   transform: rotate(270deg) scaleX(-1);
 } */
 
-.cell.flipped.rotated-0 {
+.cell.flipped {
```
Comment (vandamm) [inline src/Cell.css:60]: This commit does not belong to this pull request: it fixes different thing.

Comment (frudolph77) [reply src/Cell.css:60]: Yeah. it's from my other pull request, just messed up the branches.
